Menu/Tab/index.jsx

```diff
@@ -82,6 +84,16 @@ export default React.createClass({
     }
   },
 
+  componentWillUnmount () {
+    window.removeEventListener('resize', this.resizeListener)
+  },
+
+  resizeListener () {
+    window.requestAnimationFrame
+      ? window.requestAnimationFrame(() => update(this))
```

Comment:
This would call `requestAnimationFrame` on every `resize` callback, so I assume it would queue potentially _lots_ of update calls?